controller/src/main.cpp

```diff
@@ -60,6 +61,31 @@ static ControllerStatus controller_status = ControllerStatus_init_zero;
 // Last-received status from the GUI.
 static GuiStatus gui_status = GuiStatus_init_zero;
 
+// desired actuators state
+static ActuatorsState actuators_state;
+
+class Controller {
+public:
+  ActuatorsState Run(Time now, VentParams params, SensorReadings readings) {
+    // TODO: Add additional fields to the ControllerStatus proto:
+    //   - Blower speed (0-255 PWM value)
+    //   - Expiratory solenoid valve state (open/closed).
+    BlowerSystemState desired_state = blower_fsm_desired_state(now, params);
+
+    // pass setpoint to controller status for testing reasons
+    controller_status.fan_setpoint_cm_h2o =
+        desired_state.setpoint_pressure.cmH2O();
```

Comment:
Leave a TODO to move this out of ControllerStatus and put ActuatorsState into the proto?

Reply:
(I see now that you have a TODO below.  Perhaps just delete the comment if you don't want to duplicate it.)

Reply:
Indeed, I'll add fan_setpoint_cm_h2o to ActuatorState, and only leave the second comment.